demo/common/assets.js

```diff
@@ -1341,23 +1341,43 @@ shakaAssets.testAssets = [
   // }}}
 
   // MPEG-5 LCEVC assets {{{
-  /* LCEVC Enabled Content with LCEVC Encoded Stream */
+  /* LCEVC Enhanced Contents */
   new ShakaDemoAssetInfo(
-      /* name= */ 'Big Buck Bunny (LCEVC H264)',
+      /* name= */ 'Big Buck Bunny LCEVC H264 (DASH, MP4)',
       /* iconUri= */ 'https://storage.googleapis.com/shaka-asset-icons/big_buck_bunny.png',
-      /* manifestUri= */ 'https://dyctis843rxh5.cloudfront.net/vnIAZIaowG1K7qOt/master.m3u8',
+      /* manifestUri= */ 'https://d3mfda3gpj3dw1.cloudfront.net/vn9s0p86SVbJorX6/master.mpd',
       /* source= */ shakaAssets.Source.VNOVA)
-      .addFeature(shakaAssets.Feature.HLS)
+      .addFeature(shakaAssets.Feature.DASH)
       .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
       .addFeature(shakaAssets.Feature.MP4)
-      .addFeature(shakaAssets.Feature.WEBM)
       .addFeature(shakaAssets.Feature.OFFLINE)
       .addFeature(shakaAssets.Feature.LCEVC)
-      .addDescription('H264 HLS stream with LCEVC enhancement')
+      .addDescription('LCEVC-enhanced test stream. Shows moving squares when LCEVC is decoded successfully.')
       .markAsFeatured('Big Buck Bunny (LCEVC H264)')
+      .setExtraConfig({
+        lcevc: {
+          enabled: true,
+          dynamicPerformanceScaling: true,
+          logLevel: 0,
+          drawLogo: false,
+        },
+      }),
+  new ShakaDemoAssetInfo(
+      /* name= */ 'eSports LCEVC H264 (HLS, TS)',
+      /* iconUri= */ 'https://storage.googleapis.com/shaka-asset-icons/shaka.png',
+      /* manifestUri= */ 'https://d3mfda3gpj3dw1.cloudfront.net/vnmITf0oAwlErGf9/master.m3u8',
+      /* source= */ shakaAssets.Source.VNOVA)
+      .addFeature(shakaAssets.Feature.HLS)
+      .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
+      .addFeature(shakaAssets.Feature.MP2TS)
+      .addFeature(shakaAssets.Feature.OFFLINE)
+      .addFeature(shakaAssets.Feature.LCEVC)
+      .addDescription('LCEVC Enhanced eSports content selection.')
```

Comment:
You only need to use `addDescription` for assets that have had `markAsFeatured` called on them. The description does nothing if the asset isn't on the home page.

Reply:
Done!